pcdsdevices/dream_motion.py

```diff
@@ -0,0 +1,44 @@
+"""
+DREAM Motion Classes
+This module contains classes related to the TMO-DREAM Motion System
+
+"""
+
+from ophyd import Component as Cpt
+
+from .device import GroupDevice
+from .epics_motor import BeckhoffAxis
+from .interface import BaseInterface
+
+
+class DREAM(BaseInterface, GroupDevice):
+    """
+    DREAM Motion Class
+    This class controls motors fixed to the DREAM Motion system for the IP1
+    endstation in TMO. It also controls LED rings for the endstation.
+    Parameters
+    ----------
+    prefix : str
+        Base PV for the DREAM motion system
+    desc : str
+        Description field for LED.
+    pct : str
+        Illumination percentage of a particular endstation LED.
+    pwr : str
+        ON/OFF powered boolean of a particular endstation LED.
+    name : str
+        Alias for the device
+    """
```

Comment:
Hi Jing, maybe most of this documentation isn't correct?

Everything else looks good but maybe we should remove the parts that are not true.

For example, DREAM is not IP1, this class doesn't have LEDs, etc.

```suggestion
    This class controls motors fixed to the DREAM Motion system for the
    DREAM endstation in TMO.

    Parameters
    ----------
    prefix : str
        Base PV for the DREAM motion system
    name : str, keyword-only
        Alias for the device
    """
```